resources/views/default-master.twig

```diff
@@ -42,9 +42,11 @@
     <body class="{{ cssClasses|join(' ') }}">
         <div id="page" class="page">
             {%- if header -%}
-                <div id="themeHeader">{{ header|raw }}</div>
+                <noscript id="themeHeader">
+                    {{ header|raw }}
+                </noscript>
             {%- endif -%}
-            <div id="vanillaHeader"></div>
+            <header id="vanillaHeader"></header>
```

Comment:
Don't put "header" or "footer" tags for the client. As covineant as it may seem, many clients want to drop in their headers/footers without having to touch the HTML/CSS. It's likely to cause duplicate header/footer tags.

Reply:
We could, however, find an easy way to include it, or detect if it exists in the client's HTML.

Reply:
This our own header. I've removed the header inside that was created by react, because the extra level of nesting was causing `position: sticky` to fail and `display: contents` is not widely supported yet.

Reply:
End result for example
![image](https://user-images.githubusercontent.com/1770056/54135067-83439580-43ef-11e9-9f73-fc592de54f47.png)